packages/discovery-provider/src/queries/search_es.py

```diff
@@ -438,52 +516,127 @@ def user_dsl(search_str, current_user_id, must_saved=False):
     if current_user_id:
         dsl["should"].append(be_followed(current_user_id))
 
-    return default_function_score(dsl, "follower_count")
+    return {
+        "query": {
+            "function_score": {
+                "query": {"bool": dsl},
+                "functions": [
+                    {
+                        "filter": {"term": {"is_verified": True}},
+                        "field_value_factor": {
+                            "field": "follower_count",
+                            "factor": 10,
```

Comment:
Interesting how we apply much stronger boost for verified follows. Prob helps with spammy accounts with lots of followers?

Reply:
yeaa i just found better results if i separate verified and non-verified users in this case. this function score is basically multiplying relevancy scores based on follower count so it helps new verified users more.